extras/porting/linux/linux_yaml_config.cpp

```diff
@@ -1668,11 +1664,35 @@ void Supla::LinuxYamlConfig::addCommonParametersParsed(
 }
 
 void Supla::LinuxYamlConfig::addCommonParameters(const YAML::Node& ch,
-                            Supla::Element* element,
-                            int *paramCount) {
+                                                 Supla::Element* element,
+                                                 int* paramCount) {
   if (ch[Supla::InitialCaption]) {
     (*paramCount)++;
     element->setInitialCaption(
         ch[Supla::InitialCaption].as<std::string>().c_str());
   }
 }
+
+std::variant<int, bool, std::string> Supla::LinuxYamlConfig::parseStateValue(
+    const YAML::Node& node) {
+  try {
+    return node.as<int>();
+  } catch (...) {
+  }
+  try {
+    std::string strVal = node.as<std::string>();
+    std::string lowerStr = strVal;
+    std::transform(
+        lowerStr.begin(), lowerStr.end(), lowerStr.begin(), ::tolower);
+    if (lowerStr == "true" || lowerStr == "false") {
```

Comment:
I added "state on values" [1, true, "ON", "TRUE"] and last "TRUE" is not working. Probably because of this exception on this line.

Reply:
OK, corrected code sent.